apps/address-book/app/components/App/Entities.js

```diff
@@ -46,6 +46,8 @@ const Entities = ({ entities, onRemoveEntity }) => {
           <LocalIdentityBadge
             key={entryAddress}
             entity={entryAddress}
+            shorten={true}
```

Comment:
We don't need the `={true}` here. If the prop `shorten` is given without a value, React takes the `true` as implicit.